.github/workflows/ci.yml

```diff
@@ -295,6 +295,14 @@ jobs:
       - name: cargo test compile
```

Comment:
There's a comment a line above which we should delete.

Reply:
Deleted in https://github.com/linebender/vello/pull/872/commits/dea6c9a761ac3bec2fbed2525eab8c2f69d89b81